examples/web-app/src/components/Home/Home.js

```diff
@@ -0,0 +1,196 @@
+import React, { useState, useEffect } from "react";
+import { UserFactory } from "nightfall-sdk";
+import "./Home.css";
+
+export default function User() {
+  const [userAddress, setUserAddress] = useState();
```

Comment:
On `accountsChanged` we call `setUserAddress`, but we never actually use `userAddress` from the state. That makes me think if we need it at all, probably not.

Reply:
Yes, I used to use it but now i have it in the Navbar.js. Spot on!